tests/unit/test_autoclaved.py

```diff
@@ -0,0 +1,62 @@
+import hashlib
+from measurements.pages import decompress_autoclaved
+
+# The s3 URL is something like:
+# https://s3.amazonaws.com/ooni-public/sanitised/2017-07-08/20170707T051609Z-IR-AS44244-web_connectivity-20170707T051545Z_AS44244_ujwFYcBJXcL2MjZnHXhBqVEOG2iHR0AuPUHB7aGGqUhdtvz70h-0.2.0-probe.json
+
+AUTOCLAVED = {
+    # This is the case of a report spanning many frames
+    # XXX this report is actually quite big maybe we want to pick as a test
```

Comment:
> quite big

I think that downloding blobs on demand is probably better than placing them into the repo.